# Microbiome of two predominant seagrass species of the Kenyan coast, Enhalus acoroides and Thalassodendron ciliatum

## Introduction
Around the world, coral reefs and seagrass beds protect shorelines and host highly productive and diverse ecosystems. They provide hatcheries for and sustain marine species, including those that we harvest for food. In Kenya, coral reefs cover approximately 600km2 along the coast and are often associated with seagrass beds, although the latter’s total coverage has not been determined. Seagrass communities are subject to frequent anthropogenic and natural disturbances. Exposure to overexploitation, pollution and climate change can lead to alterations in vegetation complexity, which in turn may affect associated fauna and microorganisms. Documented recent seagrass losses along most of the Kenyan coast has been ascribed to extensive grazing by sea urchins (Tripneustes gratilla) and this has caused habitat fragmentation and defoliated beds (Mutisia, 2009). Limited transplantation projects were undertaken to counter this trend and support natural recovery and (Mutisia 2009) showed that defoliated seagrass beds could recover fully in terms of the density, diversity and community structure of its meiofauna (specifically harpacticoid copepods).

Recent studies have shown that, when disruptions occur in an ecosystem, associated changes in the microbial communities, or microbiomes, can be at least equally or more significant than that of the macrofauna and -flora. Documenting and understanding such shifts in the composition and abundance of individual species within a microbiome, can provide insights and likely identify key microorganisms, of which the presence and abundance act as indicators of the environment’s health (refs on other relevant microbiome studies, also on seagrass if available). Following this line of reasoning, (Uku et al., 2007) assessed the presence and abundance of prokaryotic epiphytes on leaves of three seagrass species in Kenya, Thalassodendron ciliatum, Thalassia hemprichii and Cymodocea rotundata and how they varied between sites containing differing levels of nutrients, associated with human activities, in the water, using denaturing gradient gel electrophoresis (DGGE) and PCR amplified 16S rRNA gene fragments. They found higher epiphytic coverage of seagrass leaves was associated with water with more nutrients, while the microbial diversity was linked to seagrass species rather than the study sites. 

Metagenomics studies have reported on the complexity of microbiomes associated with seagrass (Cúcio, et al., 2016), and how they can be used as pointers and drivers of the biogeochemical environment within biofilms, such as those associated with many marine organisms. Knowledge of the microbial composition of these communities and how they fluctuate with changes in the environment such can provide critical insights into the sustainable use and conservation of this resource. 
On the Kenyan coast, pollution, over-exploitation of marine resources and minimal efforts towards and enforcement of conservation laws of marine environments, have caused degradation and defoliation of seagrass habitats. Recent conservation activities in Kenya focused mainly on coral reefs and mangrove forest with little direct action taken to conserve seagrass meadows, despite the fact that they are valued 3 times higher than coral reefs and 10 times more than tropical forest. 
Since little is known about the microbes associated with seagrass species in Kenya, this study aimed to characterize the genetic diversity of the microbiomes of two seagrass species that occur in Kenya, Enhalus acoroides which is eaten as a snack in Lamu and Thalassodendron ciliatum (the most predominant occurring species).

## Materials & Methods
### Seagrass sampling and DNA isolation
Baobab beach (3.629383S, 39.872271) is a public beach and Kuruwitu Marine Sanctuary (3.808319S, 39.831288E) is a protected reef in Kilifi, Kenya with extensive seagrass beds of T. ciliatum at Baobab beach and E. acoroides from Kuruwitu Marine Sanctuary. Two replicate microbiome samples were collected from leaves, roots and soil (rhizobiome) and water columns of each species. DNA extraction was done with a Powersoil DNA extraction kit (MoBio Laboratories) for sediment, leaf and root microbiomes as recommended by Ettinger, et al. (2014) and a Powerwater DNA extraction kit (MoBio Laboratories) was used for microbes collected onto 0.22m filter membranes from water samples. 

### 16S rRNA gene amplicon sequencing
The Ion 16S Metagenomics kit (ThermoFisher Scientific) with primer set V2-4-8 was used to amplify the 16S variable regions for all samples according to the kit instructions. PCR products were purified with an AMPure magnetic bead purification system (Beckman Coulter) and quantified with the QuBit dsDNA HS Assay kit and Fluorometer (ThermoFisher Scientific). Libraries were synthesized and barcode adaptors added with the Ion Plus™ Fragment Library Kit (ThermoFisher Scientific). Each barcoded library was assessed to confirm the ligation of adapters (Agilent Bioanalyzer, Agilent) and concentration of libraries (Ion Library TaqMan® Quantitation Kit, ThermoFisher Scientific). Barcoded libraries were pooled in equimolar amounts and one positive control (Microbial Mock Community A) added to one batch. Templating of the pooled library was done using the Ion PGM™ Hi-Q™ OT2 Kit on the OneTouch2™ system as per the manufacturer’s recommendation (ThermoFisher Scientific). Sequencing of 400bp fragments was performed on a 318 v2 chip using an Ion PGM and Hi-Q™ Sequencing Kit (ThermoFisher Scientific). Base calling and run demultiplexing was performed with Torrent_Suite software version 5.0.4 with default parameters for the General Sequencing application. 

Sequence processing and taxonomic assignment
Individual sequence reads were filtered to remove low quality and polyclonal sequences. Sequences matching the PGM 3′ adaptor were automatically trimmed. All PGM quality-approved, trimmed and filtered data were exported as fastq files.

1.	Perform quality control and trimming of data.
2.	Analyse data using Qiime package version 1 using an open-reference OTU picking protocol by searching reads against the Greengenes database. Generate and OTU table for use in Phyloseq (https://joey711.github.io/phyloseq/)
3.	Use the R package Phyloseq to further analyse the data generated using Qiime.

## References
1.	Mutisia, L.D. Restoration of Kenyan seagrass beds: a functional study of the associated fauna and flora. 2009. Thesis Vrije Universiteit Brussel. http://hdl.handle.net/1834/7788.
2.	Uku, J., Björk, M., Bergman, B., & Díez, B. (2007). Characterization and comparison of prokaryotic epiphytes associated with three East African seagrasses. Journal of Phycology, 43(4), 768–779. http://doi.org/10.1111/j.1529-8817.2007.00371.x.
3.	Caporaso JG, Kuczynski J, Stombaugh J, Bittinger K, Bushman FD, et al. (2010) QIIME allows analysis of high-throughput community sequencing data. Nat Methods 7: 335–336.
4.	McMurdie PJ, Holmes S (2013) phyloseq: An R Package for Reproducible Interactive Analysis and Graphics of Microbiome Census Data. PLoS ONE 8(4):e61217. doi:10.1371/journal.pone.0061217
5.	Cúcio, C., Engelen, A. H., Costa, R., & Muyzer, G. (2016). Rhizosphere microbiomes of European + seagrasses are selected by the plant, but are not species specific. Frontiers in Microbiology, 7(MAR), 1–15. http://doi.org/10.3389/fmicb.2016.00440.
